src/applications/check-in/pre-check-in/pages/Introduction/index.jsx

```diff
@@ -39,6 +40,9 @@ const Introduction = props => {
 
   useEffect(
     () => {
+      const setDataToState = async data => {
```

Comment:
This is not super related to this PR, but it was causing a react error, as it was previously trying to run this code after this page / component unmounts.

Reply:
nice!